Sample-01/Dockerfile

```diff
@@ -7,31 +7,34 @@ RUN mkdir -p /app
 WORKDIR /app
 
 COPY package.json .
+COPY package-lock.json .
 
 RUN npm install
 
 COPY . .
 
-RUN npm run build
+RUN npm run build -- --prod
```

Comment:
what's the reason to do this for running the sample?

Reply:
What's the reason for **not** doing this for running the sample? :)

It should give us a more optimized build, and will also remove the warnings in the browser console that you get when running the debug version.